transformations/americanize_britishize_english/transformation.py

```diff
@@ -0,0 +1,132 @@
+    british_american_dict = requests.get(url).json()
+
+    final_dict = {**american_british_dict, **british_american_dict, **custom_vocab}
+
+    string = " ".join([final_dict.get(word, word) for word in string.split()])
+    return string
+
+
+class AmericanizeBritishizeEnglish(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION, TaskType.SENTIMENT_ANALYSIS]
+    languages = ["en"]
+
+    def __init__(self, n=1, seed=0, max_outputs=1):
+        super().__init__(seed, max_outputs=max_outputs)
+
+        # Creating a custom vocab dictionary consisting of totally different words for same context
+        self.difference_british_to_american = {
+            "trousers": "pants",
+            "flat": "apartment",
+            "bonnet": "hood",
+            "boot": "trunk",
+            "lorry": "truck",
+            "university": "college",
+            "holiday": "vacation",
+            "jumper": "sweater",
+            "trainers": "sneakers",
+            "postbox": "mailbox",
+            "biscuit": "cookie",
+            "chemist": "drugstore",
+            "shop": "store",
+            "football": "soccer",
+            "autumn": "fall",
+            "barrister": "attorney",
+            "bill": "check",
+            "caravan": "trailer",
+            "cupboard": "closet",
+            "diversion": "detour",
+            "dustbin": "trash can",
+            "jug": "pitcher",
+            "lift": "elevator",
+            "mad": "crazy",
+            "maize": "corn",
+            "maths": "math",
+            "motorbike": "motorcycle",
+            "motorway": "freeway",
+            "nappy": "diaper",
+            "pavement": "sidewalk",
+            "post": "mail",
+            "postman": "mailman",
+            "pub": "bar",
+            "rubber": "eraser",
+            "solicitor": "attorney",
+            "tax": "cab",
+            "timetable": "schedule",
+            "torch": "flashlight",
+            "waistcoat": "vest",
+            "windscreen": "windshield",
+            "angry": "mad",
+            "caretaker": "janitor",
+            "cot": "crib",
+            "curtains": "drapes",
+            "engine": "motor",
+            "garden": "yard",
+            "handbag": "purse",
+            "hoarding": "billboard",
+            "ill": "sick",
+            "interval": "intermission",
+            "luggage": "baggage",
+            "nowhere": "noplace",
+            "optician": "optometrist",
+            "queue": "line",
+            "rubbish": "trash",
+        }
+
+    def generate(self, sentence: str):
+
+        # Replacing the keys with values and vice versa for the custom vocab dictionary
+        # And merging both of them
+        vocab_diff = dict(
+            (v, k) for k, v in self.difference_british_to_american.items()
+        )
+        vocab_diff.update(self.difference_british_to_american)
```

Comment:
Might as well make this change in the constructor itself?

Reply:
Fixed.